worker/lib/workers/qemu.ts

```diff
@@ -77,6 +77,17 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 	}
 
 	public async setup(): Promise<void> {
+		// set up a ssh daemon in the worker container, so the client can ssh into it
+		console.log(`Setting up sshd`)
+		await execProm('mkdir -p /root/.ssh');
+		await execProm('chmod 0700 /root/.ssh');
+		await execProm('passwd -u root');
+		await execProm('cat /root/id_rsa.pub > /root/.ssh/authorized_keys');
+		await execProm('echo -e "PasswordAuthentication no" >> /etc/ssh/sshd_config');
+		await execProm('echo -e "Port 22222" >> /etc/ssh/sshd_config');
+		await execProm('rc-status && touch /run/openrc/softlevel && rc-service sshd start');
+		console.log(`sshd setup done!`);
```

Comment:
Why not do this stuff in the worker Dockerfile and entry script?

Reply:
@klutchell I wanted to keep it to the qemu worker, it's not used for testbots.

Reply:
Sure but it's harmless to enable for both since we aren't exposing the port outside the container bridge anyway right?
It would be nice to keep the services as similar as possible.

Also some of the entry scripts already have conditions based on `WORKER_TYPE` which we could use if we really needed.

Reply:
Makes sense!